src/app_service/service/community/service.nim

```diff
@@ -775,6 +775,28 @@ QtObject:
         if self.communities.hasKey(settings.id):
           self.communities[settings.id].settings = settings
 
+      # Non approver requests to join for all communities
+      let nonAprrovedRequestsToJoinObj = responseObj["nonAprrovedRequestsToJoin"]
+
+      #var allRequests = initTable[string, CommunityMembershipRequestDto]()
```

Comment:
is this needed?

Reply:
no, sorry, will remove and update after the status-go merge